jarbas/frontend/elm/Reimbursement/Update.elm

```diff
@@ -414,34 +407,26 @@ loadReimbursements lang apiKey query =
             jsonQuery =
                 ( "format", "json" ) :: convertQuery query
         in
-            Http.get
-                (url "/api/reimbursement/" jsonQuery)
+            Http.get (url "/api/reimbursement/" jsonQuery)
                 (decoder lang apiKey jsonQuery)
                 |> Http.send LoadReimbursements
 
 
 {-| Convert a list of key/value query pairs to a valid URL:
 
-    >>> toUrl [ ( "year", "2016" ), ( "foo", "bar" ) ]
-    "#/year/2016"
-
-    >>> toUrl [ ( "foo", "bar" ) ]
```

Comment:
This comments were here because they are tests (elm-doc-test). Any special reason not to test the function with _invalid_ query parameters?

Is there an implicit assumption that they mean no harm so I shouldn't be worried? That could be true, this might be just my OCD hahaha…